src/subscription_manager/capabilities.py

```diff
@@ -0,0 +1,169 @@
+from __future__ import print_function, division, absolute_import
```

Comment:
Any reason for keeping this out of cache.py?

Reply:
Yeah, I had two reasons.
* We want to get rid of dependency injection and cache relies on dependency injection. I didn't want to add another class to dependency injection. :-) No code uses `from cache import ...` or `import cache ...`. If cache was removed from dependency injection, then I would like to move `ServerCache` to cache module. 
* The cache module is also mostly used for caching data sent to server and purpose of cache module is to minimize sending data to server. The purpose of capabilities is little bit different it is used to minimize getting data from server.